walkthroughSteps/step-3/database/db_connection.js

```diff
@@ -1,9 +1,6 @@
 const { Pool } = require('pg');
 const url = require('url');
 require('env2')('./config.env');
-/* require('env2')('../config.env'); =
```

Comment:
Should be added to the mentor's notes, would explain this is syntactic sugar and is only an abbreviation to the 2 lines you usually use.